android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

Comment:
Can we keep it at `7.6`

Reply:
We can't do that, Because the new version flutter (>= 3.19), must support AGP 8, and the Minimum version for the AGP version is 8.0 of Gradle, refer to https://developer.android.com/build/releases/gradle-plugin?hl=zh-cn and more detail refer to another repo https://github.com/hui-z/image_gallery_saver/pull/295, Thanks.

Reply:
The error log:

Build file '/Users/yiyue.wang/opensource/flutter_line_sdk/example/android/app/build.gradle' line: 2

* What went wrong:
An exception occurred applying plugin request [id: 'com.android.application']
> Failed to apply plugin 'com.android.internal.version-check'.
   > Minimum supported Gradle version is 8.6. Current version is 7.6. If using the gradle wrapper, try editing the distributionUrl in /Users/yiyue.wang/opensource/flutter_line_sdk/example/android/gradle/wrapper/gradle-wrapper.properties to gradle-8.6-all.zip

Reply:
@PirateBrook ,
The current reviewing file name is `android/gradle/wrapper/gradle-wrapper.properties`.
The error message you provided is about `example/android/gradle/wrapper/gradle-wrapper.properties`
I agree we can set the Gradle version in example app to `8.6`.

I tried to build this PR with the following setting and it builds successfully:
1. Flutter v3.22.2; Dart 3.4.3 (stable)
1. Gradle `7.6` in `android/gradle`
1. Gradle `8.6` in `example/android/gradle`

Could you take a look, please?
Thanks.